crane-cloud/frontend
# Ft user profile

# Description

Add Social User Profile screens (Design implementation, dummy data is used her)

## Type of change



## Trello Ticket ID

https://trello.com/c/JrlSpMsy

## How Can This Been Tested?

Run this branch and checkout 

http://localhost:3000/profile/23 , after logging in to see the dummy user data representation

## Checklist:


# Screenshots
<img width="1410" alt="Screenshot 2024-06-12 at 11 32 19" src="https://github.com/crane-cloud/frontend/assets/69305400/938075bf-bac9-4d93-8527-fef23938b3f8">
<img width="1427" alt="Screenshot 2024-06-12 at 11 32 37" src="https://github.com/crane-cloud/frontend/assets/69305400/5f26a465-4694-44e8-8659-4abbe8942a31">
<img width="1423" alt="Screenshot 2024-06-12 at 11 33 54" src="https://github.com/crane-cloud/frontend/assets/69305400/9320db54-22e7-495d-a089-4ecc36a4c302">
d
<img width="1425" alt="Screenshot 2024-06-12 at 11 34 11" src="https://github.com/crane-cloud/frontend/assets/69305400/f205f35b-3bd7-46c8-81a9-86c464f03c85">
